lib/active_record/mysql/enum/validations.rb

```diff
@@ -13,18 +13,18 @@ def validates_columns(*column_names)
           column_names.each do |name|
             col = cols[name.to_s]
             raise ArgumentError, "Cannot find column #{name}" unless col
-            
+
```

Comment:
It looks like this is a copy of the ActiveRecord method that just picks off enums? Does the rest of the method still match Rails? Could this be redone as a `prepend` module that handles `enums` and then calls `super` for any other column types?

Reply:
I was looking and I don't think rails ever had (or at least I could find no documentation around) a `validates_columns` method.  This looks to be entirely isolated to this gem.  I think we can do better here for sure, but I'd like to put some thought into it rather than blocking the initial release for backwards compatibility,

Reply:
Oh, weird! I thought that name seemed so like the rest of the Rails methods...but oddly, it wasn't one I'd heard of.

So it's non-standard... and buggy. I don't think we need to keep it since this is a new gem anyway. I do like keeping the valuable part--enum inclusion--with the bug fixed. How about a name like
```
validate_enum_inclusion column_name
```
? (Or possibly a list of columns...but I'm not sure we need that.) Later we can plumb it from HoboFields when I add in my passion project that calls validations. Some Rails examples of doing the validation longhand: https://stackoverflow.com/questions/8146965/how-do-i-specify-and-validate-an-enum-in-rails

Reply:
Yea that sounds like a great idea.  It will require a usage change in some of our usages internally in our main application, but I think that's more a reason to start making those changes now, and put them in a `1.0.0` release of the new gem